src/identity-wallet.ts

```diff
@@ -266,44 +327,86 @@ class IdentityWallet {
    * @param     {String}    space                   The space used for encryption
    * @return    {String}                            The decrypted message
    */
-  async decrypt (encObj, space, toBuffer) {
-    if (!this._keyring) throw new Error('This method can only be called after authenticate has been called')
+  async decrypt(encObj, space, toBuffer) {
+    if (!this._keyring)
+      throw new Error(
+        'This method can only be called after authenticate has been called',
+      )
 
     let paddedMsg
     if (encObj.ephemeralFrom) {
-      paddedMsg = this._keyring.asymDecrypt(encObj.ciphertext, encObj.ephemeralFrom, encObj.nonce, { space, toBuffer })
+      paddedMsg = this._keyring.asymDecrypt(
+        encObj.ciphertext,
+        encObj.ephemeralFrom,
+        encObj.nonce,
+        { space, toBuffer },
+      )
     } else {
-      paddedMsg = this._keyring.symDecrypt(encObj.ciphertext, encObj.nonce, { space, toBuffer })
+      paddedMsg = this._keyring.symDecrypt(encObj.ciphertext, encObj.nonce, {
+        space,
+        toBuffer,
+      })
     }
     if (!paddedMsg) throw new Error('IdentityWallet: Could not decrypt message')
     return toBuffer ? paddedMsg : unpad(paddedMsg)
   }
 
-  async hashDBKey (key, space) {
+  async hashDBKey(key, space) {
     const salt = this._keyring.getDBSalt(space)
     return sha256Multihash(salt + key)
   }
 
-  async _get3id (space) {
+  async _get3id(space) {
     const pubkeys = this._keyring.getPublicKeys({ space, uncompressed: true })
     const doc = new DidDocument(fakeIpfs, DID_METHOD_NAME)
     if (!space) {
       if (this.DID) return this.DID
-      doc.addPublicKey('signingKey', 'Secp256k1VerificationKey2018', 'publicKeyHex', pubkeys.signingKey)
-      doc.addPublicKey('encryptionKey', 'Curve25519EncryptionPublicKey', 'publicKeyBase64', pubkeys.asymEncryptionKey)
-      doc.addPublicKey('managementKey', 'Secp256k1VerificationKey2018', 'ethereumAddress', pubkeys.managementKey)
-      doc.addAuthentication('Secp256k1SignatureAuthentication2018', 'signingKey')
+      doc.addPublicKey(
+        'signingKey',
+        'Secp256k1VerificationKey2018',
+        'publicKeyHex',
+        pubkeys.signingKey,
+      )
+      doc.addPublicKey(
+        'encryptionKey',
+        'Curve25519EncryptionPublicKey',
+        'publicKeyBase64',
+        pubkeys.asymEncryptionKey,
+      )
```

Comment:
Is this because of prettier that the line get's too long? Seems a litte verbose?

Reply:
Yes it's prettier formatting, by default the line length is 80 characters but we can configure to a higher value if wanted?

Reply:
140? 80 seems very short.

Reply:
Sorry I missed that comment, here is an explanation for the line length option in prettier: https://prettier.io/docs/en/options.html#print-width
I've found it pretty good overall, I guess it depends on everyone's editor setup but mine would start having horizontal scroll over 110 characters so setting this config to 140 would be very annoying. Maybe we can start with 100 and see how it works for everyone?

Reply:
Sounds good @PaulLeCam 👍